src/material/list/testing/list-harness-filters.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BaseHarnessFilters} from '@angular/cdk/testing';
+
+export interface ListHarnessFilters extends BaseHarnessFilters {}
+
+export interface ActionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface NavListHarnessFilters extends BaseHarnessFilters {}
+
+export interface SelectionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface BaseListItemHarnessFilters extends BaseHarnessFilters {
+  text?: string | RegExp;
+}
+
+export interface ListItemHarnessFilters extends BaseListItemHarnessFilters {}
+
+export interface ActionListItemHarnessFilters extends BaseListItemHarnessFilters {}
+
+export interface NavListItemHarnessFilters extends BaseListItemHarnessFilters {
+  href?: string | RegExp | null;
```

Comment:
Should we drop the `null` here or is it there so consumers can look up items without an `href`?

Reply:
Yeah `href=""` is not the same as missing the `href` attr